dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -651,17 +651,23 @@ def stub_country(code)
     toolbox_translated_name = "Azioni"
     @level.toolbox_blocks = toolbox
 
-    start = "<xml><block type=\"procedures_defnoreturn\"><title name=\"NAME\">details</title></block></xml>"
+    start = "<xml><block type=\"procedures_defnoreturn\"><mutation><description>function description</description></mutation><title name=\"NAME\">details</title></block></xml>"
```

Comment:
I'd love to see a test that works with either just name or name+description, to confirm that the minimal case still works

Reply:
This one works for name+description and there is a different test in this file that only translates the name (though there's not description in that block). I can add more tests that test different combinations, including fallback.

Reply:
oh, sorry, to be more clear: I want to see a test that operates over a block that only _defines_ the name and not the description. Just to make sure the i18n logic doesn't panic when given minimal XML

Reply:
looking at the other test you mention, I think it does just want I want!